### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by mbstrange2**
Add torch to dockerfile for garnet updates

### CODE REVIEW

**Review by steveri — APPROVED** [review]
LGTM as long as all the checks pass...

**Comment by mbstrange2** [comment]
@steveri It failed due to running out of space...any idea how to fix that?

**Comment by steveri** [comment]
> @steveri It failed due to running out of space...any idea how to fix that?

Which test are you referring to? For buildkite/aha-flow, I see this error...
```
The command '/bin/bash --login -c mkdir /aha/tmp/torch_install/' returned a non-zero code: 1
```
...which I think can be fixed by adding `-p` to the mkdir, i.e. changing the command from 
```
mkdir /aha/tmp/torch_install/
```
to
```
mkdir -p /aha/tmp/torch_install/
```

File: Dockerfile [hunk 1]
```diff
@@ -113,6 +113,11 @@ WORKDIR /aha/sam
 RUN make sam
 RUN source /aha/bin/activate && pip install scipy numpy pytest && pip install -e .
 
+# Install torch (need big tmp folder)
+WORKDIR /aha
+RUN mkdir /aha/tmp/torch_install/
```
Comment (steveri) [inline Dockerfile:118]: Maybe "mkdir" here should instead be "mkdir -p"

**Comment by steveri** [comment]
Ah. I see it now.
```
ApplyLayer exit status 1 stdout:  stderr: write /aha/lib/python3.8/site-packages/torch/lib/libtorch_cuda.so: no space left on device
```
This runs on github's own VM, yes? Not sure what we can do about more disk space? How big is torch, anyway? And/or what else can we do to reduce the build footprint?

**Comment by mbstrange2** [comment]
> Ah. I see it now.
> 
> ```
> ApplyLayer exit status 1 stdout:  stderr: write /aha/lib/python3.8/site-packages/torch/lib/libtorch_cuda.so: no space left on device
> ```
> 
> This runs on github's own VM, yes? Not sure what we can do about more disk space? How big is torch, anyway? And/or what else can we do to reduce the build footprint?

Turns out there's a cpu-only version of torch that seems to work.

**Pull request merged**